### PR TITLE
Manage copyright from Mapbox layer

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: src/layers/MapboxLayer.js [hunk 1]
```diff
@@ -6,6 +6,21 @@ import OLLayer from 'ol/layer/Layer';
 import GeoJSON from 'ol/format/GeoJSON';
 import Layer from './Layer';
 
+const getCopyrightFromSources = mbMap => {
+  let copyrights = [];
+  const regex = /<[^>]*>[^>]*<\/[^>]*>/g;
+  // Trick from Mapbox AttributionControl to know if the source is used.
+  const { sourceCaches } = mbMap.style;
+  Object.entries(sourceCaches).forEach(([, sourceCache]) => {
+    if (sourceCache.used) {
```
Comment (gevos) [inline src/layers/MapboxLayer.js]: In this example, `sourceCache.used` is false. Is this intended?

File: src/layers/MapboxLayer.js [hunk 2]
```diff
@@ -6,6 +6,21 @@ import OLLayer from 'ol/layer/Layer';
...
+      copyrights = copyrights.concat(
+        regex.exec(sourceCache.getSource().attribution),
+      );
+    }
+  });
+  return [...new Set(copyrights.filter(copyright => !!copyright))].join(', ');
```
[the remark this answers is not in the document]
Comment (oterral) [reply src/layers/MapboxLayer.js]: [..newSet] is the shorter version

File: src/layers/MapboxLayer.js (hunk 1 above)
Comment (oterral) [reply src/layers/MapboxLayer.js]: We display the source's copyright only if the source is used by the style.

File: src/layers/MapboxLayer.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (gevos) [reply src/layers/MapboxLayer.js]: It didn't work for me.

[the remark this answers is not in the document]
Comment (oterral) [reply src/layers/MapboxLayer.js]: I jus t tested it un the console it works : [...new Set(['lala', 'lala'])]  => ['lala'']

[the remark this answers is not in the document]
Comment (gevos) [reply src/layers/MapboxLayer.js]: @oterral But `[...new Set()]` is compiled to `[].concat[new Set()]`. And this returns `[object Set]`.

[the remark this answers is not in the document]
Comment (oterral) [reply src/layers/MapboxLayer.js]: thx for the explanation. I fix.

Comment (oterral) [inline src/layers/MapboxLayer.js]:
```suggestion
  return Array.from(new Set(copyrights.filter(copyright => !!copyright))).join(', ');
```

[the remark this answers is not in the document]
Comment (oterral) [reply src/layers/MapboxLayer.js]: fixed

**Pull request merged**